pkl-core/src/test/files/LanguageSnippetTests/input/listings/listings7.pkl

```diff
@@ -0,0 +1,4 @@
+// ensure that this member is only evaluated once (should result in only one trace)
```

Comment:
nit: all other tests are in the singular (`listing4/5/6.pkl`), only this one is plural.

Reply:
Good catch!